cmd/balancesheet/balance-sheet.go

```
@@ -0,0 +1,339 @@
+	cmd.Flags().StringP("type", "t", "balance", "Report type: balance, income, cashflow")
+	cmd.Flags().StringP("period", "p", "annual", "Period filter: annual, halfyear, q1, q3")
+	cmd.Flags().StringP("output", "o", "", "Export to file (.csv or .json)")
+
+	return cmd
+}
+
+// result holds fetched report data for a specific type.
+type result struct {
+	rt    eastmoney.FinancialReportType
+	items []*eastmoney.FinancialReportItem
+	err   error
```

golangci-lint (2.12.2)

[error] field err is unused

(unused)